timemachine/ff/__init__.py

```diff
@@ -88,20 +87,18 @@ def load_from_file(cls, path_or_str: Union[str, Path]) -> "Forcefield":
         original_path = str(path_or_str)
         path = Path(path_or_str)  # Safe to construct a Path object from another Path object
 
-        with resources.files("timemachine.ff.params") as params_path:
-            built_in_path = params_path / path.name
-            if built_in_path.is_file():
-                if path.is_file():
-                    warn(
-                        f"Provided path {original_path} shares name with built-in forcefield, falling back to built-in"
-                    )
-                # Search built in params for the forcefields
-                path = built_in_path
-            if not path.is_file():
-                raise ValueError(f"Unable to find {original_path} in file system or built-in forcefields")
-            with open(path, "r") as ifs:
-                handlers, protein_ff, water_ff = deserialize_handlers(ifs.read())
+        # Look for builtin ff
+        for resource in resources.files("timemachine.ff.params").iterdir():
+            if resource.name == path.name:
+                warn(f"Provided path {original_path} shares name with built-in forcefield, falling back to built-in")
+                handlers, protein_ff, water_ff = deserialize_handlers(resource.read_text())
+                return cls.from_handlers(handlers, protein_ff=protein_ff, water_ff=water_ff)
```

Comment:
```suggestion
        with resources.as_file(resources.files("timemachine.ff.params") / path.name) as rpath:
            if rpath.exists():
                warn(f"Provided path {original_path} shares name with built-in forcefield, falling back to built-in")
                handlers, protein_ff, water_ff = deserialize_handlers(rpath.read_text())
                return cls.from_handlers(handlers, protein_ff=protein_ff, water_ff=water_ff)
```

nit: use `resources.as_file` to avoid iteration?